tlmgr-check: add page (#6586)
tldr-pages/tldr: pages/common/tlmgr-check.md

# tlmgr check

> Check the consistency of a TeX Live installation.
> More information: <https://www.tug.org/texlive/tlmgr.html>.

> - Check the consistency of the whole TeX Live installation:

Reviewer: ```suggestion
- Check the consistency of the whole installation:
```

Author: Why do you think it should be removed? The sentence is quite short and more explicit is always better.

Reviewer: > more explicit is always better

Do we have any rule that states that? Because I disagree a lot. There are many horrible ways to make this a lot more explicit about everything. The description explicitly mentions that this operates on a TeX Live installation, so this isn't just explicit, it is redundant.

Author: We do have a rule which states that examples should not depend on the others and can be read on it's own. @CleanMachine1  / @navarroaxel wdyt?

Reviewer 2: In this case I think mention `TeX Live` is better.

Reviewer 3: I think keeping this is best, because it's more explicit. Otherwise, one would question which executables it checks.

Reviewer 4: Its fine to me. Same opinion as others thoughts.

`tlmgr check all`

> - Check the consistency of the whole TeX Live information in verbose mode:

Reviewer: ```suggestion
- Check the consistency of the whole installation in verbose mode:
```

`tlmgr check all -v`

> - Check for not installed dependencies:

Reviewer: ```suggestion
- Check for missing dependencies:
```

`tlmgr check depends`

> - Check if all TeX Live executables are present:

Reviewer: ```suggestion
- Check if all executables are present:
```

Reviewer 4: Explicit here is definitely better.

`tlmgr check executes`

- Check if all files listed in the local TLPDB are present:

`tlmgr check files`

> - Check for duplicate filenames in the runfiles sections:

Reviewer 4: ```suggestion
- Check for duplicate filenames in the runfiles section:
```

Reviewer 4: Does this need to be plural?

If not then merge (I've approved)

Author: That's what the man page says, so I'll just rely on that
